aiida_pseudo/cli/install.py

```diff
@@ -140,3 +140,111 @@ def cmd_install_sssp(version, functional, protocol, traceback):
         family.set_cutoffs(cutoffs)
 
         echo.echo_success(f'installed `{label}` containing {family.count()} pseudo potentials')
+
+
+@cmd_install.command('pseudodojo')
+@options.VERSION(type=click.Choice(['03', '04']), default='04')
+@options.FUNCTIONAL(type=click.Choice(['pbe', 'pbesol', 'lda']), default='pbe')
+@options.REL(type=click.Choice(['sr', 'sr3plus', 'fr']), default='sr')
+@options.PROTOCOL(type=click.Choice(['standard', 'stringent']), default='standard')
+@options.HINT(type=click.Choice(['low', 'normal', 'high', 'none']), default='none')
```

Comment:
What is the difference between the protocol and hint? Why does the protocol not also determine what cutoffs should be used with it?

Reply:
The protocol corresponds to parameters used by the code that generates the pseudopotentials (I think it's GBRV or something). PseudoDojo makes a distinction between this (at the pseudopotential generation level) and the hints, which they determine based on different convergence criteria.

As far as I can tell, the protocol -> a set of pseudopotentials, while the hints -> a set of convergence criteria applied to the set of pseudopotentials which come from the protocol.

Reply:
This is correct but I can elaborate a bit on this. Basically 'standard' psp are generated with the goal of doing DFT or DFPT calculations. For each psp, a set of energy ecut is provided if you want low, normal or high accuracy. 
In the case of many body (like the GW method), you need pseudopotential with more semicore states and more stringent parameters (the cut-off radii beyond which the psp is equal to the all-electron wavefunctions is shorter). 
In this case they generated the 'stringent' set of psp. Then again for that set you have suggested low, normal or high accuracy ecut. 
Hope this clarifies it.

Reply:
I chose the word protocol, but it is not really a term the SSSP paper uses as well. They don't have a term, but I needed one for the plugin. I chose it to distinguish between the two versions they have, the "efficiency" and "precision". Each of these families come with a different set of pseudos (some overlap exists) and an individual set of recommended cutoffs for each set of pseudos. I think the term protocol makes sense, also in the sense we are using it of the `aiida-common-workflows` project. The values of PD `stringent` and `standard` also nicely fall onto that definition, so I think it is fine to keep it. Regarding "hint", I think that term is too general and  vague. Given that in both cases that this applies, both PD as well as the SSSP, it solely is considered the recommended cutoffs, maybe we should be explicit and call it "stringency" and so make the reusable command line option `options.CUTOFF_STRINGENCY('--cutoff-stringency')`. I think that is clear